model/renderNativeWindow.cc

```diff
@@ -41,17 +40,22 @@ namespace minsky
 {
   void RenderNativeWindow::renderFrame(unsigned long parentWindowId, int offsetLeft, int offsetTop, int childWidth, int childHeight)
   {
-    if (!winInfoPtr.get())
-     winInfoPtr=std::make_shared<WindowInformation>(parentWindowId, offsetLeft, offsetTop, childWidth, childHeight);
+    bool isInitial = false;
```

Comment:
What is this flag used for?

Reply:
You should have received a note from Travis about the build failing. It turns out the compiler is asking the same question :).

Reply:
@highperformancecoder   Oh yes, sorry about that. Commented it out. That block of code may go away finally but leaving for now in case some optimization is required/possible on other platforms.